examples/model_selection/plot_multi_metric_evaluation.py

```diff
@@ -0,0 +1,85 @@
+"""Demonstration of multi-metric evaluation on cross_val_score and GridSearchCV
+
+Multiple metric grid search (or random search) can be done by setting the
+``scoring`` parameter to a list of metric scorer names or a dict mapping the
+scorer names to the scorer callables.
+
+The scores of all the scorers are available in the ``cv_results_`` dict at keys
+ending in ``'_<scorer_name>'`` (``'mean_test_precision'``,
+``'rank_test_precision'``, etc...)
```

Comment:
since when do we have rank? Maybe say ``mean_test_roc_auc``, ``mean_test_f1`` as those are the one that are actually used below.